plugin/documents.py

```diff
@@ -139,6 +139,20 @@ class DocumentSyncListener(sublime_plugin.ViewEventListener, AbstractViewListene
     highlights_debounce_time = FEATURES_TIMEOUT
     code_lenses_debounce_time = FEATURES_TIMEOUT + 2000
 
+    ANNOTATIONS_TEMPLATE = """
+    <body id="lsp-annotation">
+        <style>
+        html, body {{
+            padding: 0;
+            margin: 0;
+            font-family: system;
+            height:{height}
```

Comment:
I don't think this does anything. `view.line_height()` is in pixels and you don't have the unit here.
But if it was meant to fix the https://github.com/sublimelsp/LSP/pull/1663#discussion_r632047619 issue then it doesn't succeed in any case.